src/Bridge/Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -353,6 +353,10 @@ private function shouldDoctrinePaginatorUseOutputWalkers(QueryBuilder $queryBuil
             if (null !== $useOutputWalkers = $resourceMetadata->getCollectionOperationAttribute($operationName, 'pagination_use_output_walkers', null, true)) {
```

Comment:
We should check if it's a subresource above and then do the right call accordingly